tasks/create_deploy_user.yml

```diff
@@ -17,6 +17,4 @@
     path: "{{ getent_passwd[django_deployment_user][4] }}"
     state: directory
     mode: "u+rx,g+rx"
-  when: (check_deploy_user_getent|succeeded) and (ansible_distribution=="Ubuntu")
-
-  
\ No newline at end of file
+  when: (check_deploy_user_getent is succeeded) and (ansible_distribution=="Ubuntu")
```

Comment:
nitpic: `when` on various lines are combined with `and`. This line is equivalent to

```
when: 
 - check_deploy_user_getent is succeeded
 - ansible_distribution=="Ubuntu"
```